### DISCUSSION TITLE
How to build turborepo app using  AWS Sam build?

### DISCUSSION THREAD

**Discussion by fnLog0** [vercel/turborepo#7345] [Help]
I have a TurboRepo for an AWS AppSync application, and I want to deploy it using AWS SAM. However, when I run `sam build`, it's using local packages from npm instead of taking them from the workspace. Can you please assist me with this?

**Comment by ryantando** [comment]
@fnLog0 have you found the solution?

**Reply by fnLog0** [reply]
I've migrated to a serverless setup for Apollo Server
